pkg/tenant/db/members.go

```diff
@@ -89,10 +98,6 @@ func (m *Member) Validate() error {
 		return ErrUnknownMemberRole
 	}
 
-	if m.Status == "" {
-		return ErrMissingMemberStatus
-	}
```

Comment:
With the new constants, the default value (0) is `Pending`. We could have also made this `Unknown`, and still can, but I thought the default made sense.